atd-vze/src/views/Crashes/Maps/CrashEditCoordsMap.js

```diff
@@ -1,189 +1,130 @@
-      markerLongitude: updatedViewport.longitude,
-    });
-  };
 
-  handleMapFormCancel = e => {
-    e.preventDefault();
-    this.props.setIsEditingCoords(false);
+    // Move map center and marks to original coordinates or default fallback
+    setMarkerCoordinates(originalMarkerCoordinates);
+    mapRef.current &&
+      mapRef.current.jumpTo({
+        center: [
+          originalMarkerCoordinates.longitude,
+          originalMarkerCoordinates.latitude,
+        ],
+      });
   };
 
-  render() {
-    const {
-      viewport,
-      markerLatitude,
-      markerLongitude,
-      isDragging,
-    } = this.state;
-    const geocoderAddress = this.props.mapGeocoderAddress;
-    const isDev = window.location.hostname === "localhost";
+  const handleFormCancel = () => {
+    setIsEditingCoords(false);
+  };
 
-    return (
-      <div>
-        <MapGL
-          {...viewport}
-          ref={this.mapRef}
-          width="100%"
-          height="350px"
-          mapStyle={
-            isDev
-              ? "mapbox://styles/mapbox/satellite-streets-v11"
-              : LOCATION_MAP_CONFIG.mapStyle
-          }
-          onViewportChange={this._updateViewport}
-          getCursor={this.getCursor}
-          // controller={customGeocoderMapController}
-          mapboxApiAccessToken={TOKEN}
-        >
-          {/* <Geocoder
-            mapRef={this.mapRef}
-            onViewportChange={this._handleViewportChange}
-            mapboxApiAccessToken={TOKEN}
-            inputValue={geocoderAddress}
-            options={{ flyTo: false }}
-            // Bounding box for auto-populated results in the search bar
-            bbox={[-98.22464, 29.959694, -97.226257, 30.687526]}
-          /> */}
-          <div className="fullscreen" style={fullscreenControlStyle}>
-            <FullscreenControl />
-          </div>
-          <div className="nav" style={navStyle}>
-            <NavigationControl showCompass={false} />
-          </div>
-          {/* add nearmap raster source and style */}
-          {!isDev && <LabeledAerialSourceAndLayer />}
-          <Marker latitude={markerLatitude} longitude={markerLongitude}>
-            <Pin size={40} isDragging={isDragging} animated />
-          </Marker>
-        </MapGL>
-        <CrashEditLatLonForm
-          latitude={markerLatitude}
-          longitude={markerLongitude}
-          handleFormSubmit={this.handleMapFormSubmit}
-          handleFormReset={this.handleMapFormReset}
-          handleFormCancel={this.handleMapFormCancel}
-        />
-      </div>
-    );
-  }
-}
+  return (
+    <>
+      <MapGL
+        ref={mapRef}
+        initialViewState={{
+          latitude: latitude_primary || defaultInitialState.latitude,
+          longitude: longitude_primary || defaultInitialState.longitude,
+          zoom: defaultInitialState.zoom,
+        }}
+        style={{ width: "100%", height: "50vh" }}
+        {...mapParameters}
+        cooperativeGestures={true}
```

Comment:
i think we might want to leave this this off (`false`) when editing the coordinates. going on the assumption that you probably want scroll capture in edit mode. does that sound right?

Reply:
that sounds right to me too, and thanks for catching. copypasta double strike! (test steps and code) 🍝